Services/CO.CDP.Localization/StaticTextResource.resx

```diff
@@ -285,9 +285,6 @@
   <data name="Organisation_MyOrganisations" xml:space="preserve">
     <value>My organisations</value>
   </data>
-  <data name="Organisation_OrganisationName" xml:space="preserve">
```

Comment:
Why have all these values been removed?

Reply:
It is not removed, I just checked and it is there.

Reply:
Ah! They were duplicated on main and you've removed the duplicates. Cool :-)